test/terra/backends/aer_simulator/test_conditional.py

```diff
@@ -78,9 +82,11 @@ def test_conditional_gates_64bit(self, method, device):
     @supported_methods(SUPPORTED_METHODS)
     def test_conditional_gates_132bit(self, method, device):
         """Test conditional gate operations on 132-bit conditional register."""
-        shots = 100
```

Comment:
I believe that this line is necessary. `shots` can be used without assignment, and then tests fail.

Reply:
I removed remaining shot branching dependencies